simple_history/models.py

```diff
@@ -448,19 +455,17 @@ def diff_against(self, old_history):
 
         changes = []
         changed_fields = []
-        instance_fields = [field.name for field in self.instance._meta.fields]
-        old_instance_fields = [
-            field.name for field in old_history.instance._meta.fields
-        ]
-        for field in self._meta.fields:
```

Comment:
I'm wondering if there's a difference between `self.instance._meta.fields` (forming `instance_fields` above) and `self._meta.fields`? If there is, I think the new code might not be iterating over the same thing.

Reply:
There is a difference. Namely in the lack of `history_*` fields, which we don't want to be diffing anyway